plugins/sigma.neo4j.bolt/README.md

```diff
@@ -0,0 +1,76 @@
+sigma.neo4j.bolt
+====================
+
+Plugin developed by [Andrea Santurbano](https://github.com/conker84).
+
+---
+
+This plugin provides connect to neo4j >= 3 via new bolt protocol using bolt javascript api (https://github.com/neo4j/neo4j-javascript-driver), parse the response and fill the graph with the `graph.read()` method.
+
+Nodes are created with the following structure :
+ * id -> Neo4j node id
+ * label -> Neo4j node id
+ * neo4j_labels -> Labels of Neo4j node
```

Comment:
I know but i have preserved the same old structure of **[sigma.neo4j.cypher](https://github.com/jacomyal/sigma.js/blob/master/plugins/sigma.neo4j.cypher/sigma.neo4j.cypher.js)** plugin.
The fields are now are camelized.